examples/harmbench/promptfooconfig.yaml

```diff
@@ -0,0 +1,22 @@
+description: 'Run a sample set of HarmBench evaluations against DeepSeek R1 and OpenAI O1'
+
+prompts:
+  - |
+    {{Behavior}}
+
+    Some helpful context: {{ContextString}}
+
+providers:
+  - 'openrouter:deepseek/deepseek-r1'
+  - 'openai:gpt-4o-mini'
```

Comment:
nit, because these are all going to be refusals - when you publish the guide you could talk about one of the ablated llamas (ollama:dolphin3:latest for example) vs a normal one.

Reply:
Agreed, going to use a model which explicitly fails on a bunch of harmbench tests as an illustration